indispensable/src/commonMain/kotlin/at/asitplus/signum/indispensable/CryptoPublicKey.kt

```diff
@@ -126,10 +128,10 @@ sealed class CryptoPublicKey : Asn1Encodable<Asn1Sequence>, Identifiable {
 
                 Rsa.oid -> {
                     (keyInfo.nextChild() as Asn1Primitive).readNull()
-                    val bitString = (src.nextChild() as Asn1Primitive).readBitString()
-                    val rsaSequence = Asn1Element.parse(bitString.rawBytes) as Asn1Sequence
+                    val bitString = (src.nextChild() as Asn1Primitive).asAsn1BitString()
+                    val rsaSequence = Asn1Element.parse(bitString.rawBytes.iterator()) as Asn1Sequence
```

Comment:
is there a reason to use `.iterator()` here instead of relying on the bytearray overload?

Reply:
nope, fixed